src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/HelpCommand.kt

```diff
@@ -0,0 +1,25 @@
+package com.noumenadigital.npl.cli.commands.registry
+
+import com.noumenadigital.npl.cli.commands.NplCliCommandsRegistry
+import java.io.Writer
+
+data class HelpCommand(
+    val params: List<String>?,
```

Comment:
Maybe the params could:

- be moved into the interface
- made into a list that's `emptyList()` by default and not nullable